### PR TITLE
Add command "Move top-level tasks in Editor to heading"

### PR DESCRIPTION
**Pull request by dwertheimer**
Added command /Move top-level tasks in Editor to heading
Added to /TidyUp
Created two settings: 
- one for tidyUp
- one for the default heading to move tasks to

### CODE REVIEW

**Comment by dwertheimer** [comment]
@jgclark the all-plugins-build CI test is failing due to:

```
Build of plugin: "/Users/runner/work/plugins/plugins/jgclark.NoteHelpers" failed Error: Could not resolve './fileNaming' from jgclark.NoteHelpers/src/index.js
```

File: np.Tidy/src/tidyHelpers.js [hunk 1]
```diff
@@ -33,6 +33,8 @@ export type TidyConfig = {
   runRemoveSectionFromNotesCommand: boolean,
   removeTriggersFromRecentCalendarNotes: boolean,
   removeTodayTagsFromCompletedTodos: boolean,
+  moveTopLevelTasksInEditor: boolean,
+  moveTopLevelTasksInEditor: string,
```
Comment (jgclark) [inline np.Tidy/src/tidyHelpers.js:37]: This looks like duplication of setting names with different types. Is that a typo?

**Review by jgclark — APPROVED** [review]
Just one small question from reading the code.

**Comment by dwertheimer** [comment]
@jgclark Please wait before you merge and release this. I may be adding a feature based on an issue I'm finding.

**Comment by dwertheimer** [comment]
merged

**Pull request merged**